Dockerfile

```diff
@@ -22,6 +22,8 @@ RUN pip install --upgrade pip && \
  make && \
  make install
 
+RUN git clone https://github.com/MycroftAI/lingua-franca.git
```

Comment:
shouldn't we use one of the forks? Carme and Joan had fixes in their forks. (probably does not affect the hour formatting)